src/olympia/devhub/urls.py

```diff
@@ -14,11 +14,9 @@
 # These will all start with /addon/<addon_id>/submit/
 submit_patterns = patterns(
     '',
-    url('^$', lambda r, addon_id: redirect('devhub.submit.6', addon_id)),
-    url('^3$', views.submit_describe, name='devhub.submit.3'),
-    url('^4$', views.submit_media, name='devhub.submit.4'),
-    url('^5$', views.submit_license, name='devhub.submit.5'),
-    url('^6$', views.submit_done, name='devhub.submit.6'),
+    url('^$', lambda r, addon_id: redirect('devhub.submit.5', addon_id)),
```

Comment:
Can you make a quick test for this, IIRC there wasn't any failure when I spotted the error earlier.